Dockerfile

```diff
@@ -12,5 +12,11 @@ ADD plugins/ ${GEOWAVE_HOME}/tools/plugins/
 ADD geowave-analytic-mapreduce.jar ${GEOWAVE_HOME}/tools/
 ADD geowave-accumulo.jar ${GEOWAVE_HOME}/accumulo/
 
+# Add a useful alias for geowave command line
+RUN echo '#!/bin/bash' > /usr/bin/geowave
+RUN echo '${GEOWAVE_HOME}/tools/geowave-tools.sh' >> /usr/bin/geowave
+RUN chmod +x /usr/bin/geowave
+
```

Comment:
Perhaps this can be turned into a file which can be `COPY`'ed (if the script is set executable on the host, that will be carried over to the container).